### ISSUE TITLE
4.5.5

### ISSUE THREAD

**Issue by mpektkd** [slp-ntua/slp-labs#54]
Καλησπερα, ειχα μια απορια σχετικα με το 4.5.5. 
![image](https://user-images.githubusercontent.com/62422421/103352076-e7948500-4aad-11eb-9c94-827ca7736ad0.png)

Εδω λεει οτι πρεπει να υπολογισουμε τις posteriors πιθανοτητες, οπου καταλαβαινω οτι πρεπει να βαλω μια συναρτηση softmax στο output layer, αλλα στο extract_posteriors.py, ζητειται η εξαγωγη των logits, που απο τι διαβασα ειναι το σταδιο της πληροφοριας πριν το softmax. Τελικα τι απο τα δυο πρεπει να εξαγω?
Επισης, αναφερεται οτι θα δωθει καποιο παραδειγμα. Γυρω απο τι?
Ευχαριστω πολυ.

**Comment by georgepar** [comment]
Τυπικά, τα posteriors είναι μετά το softmax. Το δοκιμάσαμε με softmax και χωρίς (logits) και τα logits πάνε καλύτερα, οπότε το αφήσαμε έτσι. (feel free to try it yourself).

Το παράδειγμα εννοεί τον κώδικα που δίνουμε.

Πρακτικά θέλει μια λούπα που να περνάει το dataset από την forward

**Comment by mpektkd** [comment]
Αα μαλιστα ωραια. Το decode_dnn.sh δεν θελει ομως πιθανοτητες, θελω να πω οτι αν του δωσω logits θα παιξει?

**Comment by georgepar** [comment]
Ναι θα παίξει.

**Comment by mpektkd** [comment]
Ωραια ευχαριστω πολυ. Μια τελευταια ερωτηση στο 4.5.6 τι WER περιμενουμε περιπου να δουμε μετα το decode?
